src/tasks.md

```diff
@@ -67,7 +67,7 @@ fn update(&mut self, message: Self::Message) -> cosmic::Task<cosmic::Action<Self
             self.counter += 1;
             self.counter_text = format!("Clicked {} times", self.counter);
 
-            // Await for 3 seconds in the background, and then request to decrease the counter.
+            // Wait for 3 seconds in the background, and then request to decrease the counter.
```

Comment:
This one is await

Reply:
I get that the keyword is `await`, but in this sentence I believe "wait" is the correct word.

You can await an object, but you must wait for a duration.

So the following are correct:

- I await your arrival.
- We await the future.

But this is incorrect:

- We await for 3 seconds.


However, if you would like to keep it as "Await" I'll switch it back. What do you think?